charts/app-config/values-integration.yaml

```diff
@@ -1309,6 +1309,18 @@ govukApplications:
               key: bearer_token
 
   - name: govspeak-preview
+    helmValues:
```

Comment:
**suggestion:** abbreviation

Might be worth spelling out what ALB stands for in the commit message

Reply:
Done.